### PR TITLE
Add consistent install tarballs & startup scripts

### PR DESCRIPTION
**Pull request by hansieodendaal**
## Description
This PR adds startup scripts for OSX and Ubuntu, similar to Windows, which are consistent with the updated runtime directory layout. It also adds 3rd party installation script for XMRig that will always install the latest version. Installation tarball setup
scripts for OSX and Ubuntu will prepare a new zipped tarball for release, also according to the updated directory structure. Installed artefacts will now be similar across OSX, Ubuntu and Windows. 
**Edit:** The Windows _one click miner_ is also fixed and the Windows installer updated accordingly.
**Edit:** Windows installer available [here](https://drive.google.com/drive/u/0/folders/1hIIK_Ilf6qoG4WXpsTc4BSDPPTdrmQiJ) for testing: `tari_base_node-Windows-0.6.3-4c66762-release.exe`

To prepare the installation tarball:
```
cd buildtools
create_osx_install_zip.sh <release tarball name and version>
```
or
```
cd buildtools
create_ubuntu_install_zip.sh  <release tarball name and version>
```

To install, unzip the tarball.

## Motivation and Context
Consistent user experience across all platforms, ease of use.

## How Has This Been Tested?
Tarball created on Linux, extracted, and all components run successfully.

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [X] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [X] I'm merging against the `development` branch.
* [ ] I ran `cargo-fmt --all` before pushing.
* [ ] I ran `cargo test` successfully before submitting my PR.
* [X] I have squashed my commits into a single commit.
* [X] My change requires a change to the documentation.
* [X] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,13 +26,22 @@ First you'll need to make sure you have a full development environment set up:
 #### (macOS)
 
 ```
+brew update
 brew install cmake openssl tor ncurses
```
Comment (delta1) [inline README.md:30]: testing now, macos `readlink` is BSD version, so add coreutils and change to `greadlink` 

`brew install cmake openssl tor ncurses coreutils`

File: buildtools/create_osx_install_zip.sh [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+cp -f -P "${app_dir}/tari_merge_mining_proxy/osx/start_xmrig" "${tarball_folder}/start_xmrig"
+cp -f "${app_dir}/tari_merge_mining_proxy/osx/runtime/start_tari_merge_mining_proxy.sh" "${tarball_folder}/runtime/start_tari_merge_mining_proxy.sh"
+cp -f "${app_dir}/tari_merge_mining_proxy/osx/runtime/start_xmrig.sh" "${tarball_folder}/runtime/start_xmrig.sh"
+cp -f "${project_dir}/target/release/tari_merge_mining_proxy" "${tarball_folder}/runtime/tari_merge_mining_proxy"
+
+# 3rd party install
+cp -f "${local_dir}/install_xmrig.sh" "${tarball_folder}/runtime/install_xmrig.sh"
+cp -f "${local_dir}/get_xmrig_osx.ps1" "${tarball_folder}/runtime/get_xmrig_osx.ps1"
+cp -f "${local_dir}/install_powershell_osx.sh" "${tarball_folder}/runtime/install_powershell_osx.sh"
+
```
Comment (delta1) [inline buildtools/create_osx_install_zip.sh:70]:
```zsh
cp: /Users/byron/code/tari/buildtools/get_xmrig_osx.ps1: No such file or directory
cp: /Users/byron/code/tari/buildtools/install_powershell_osx.sh: No such file or directory
```

Comment (delta1) [reply buildtools/create_osx_install_zip.sh:70]: those 2 osx scripts are missing

**Review by delta1 — CHANGES_REQUESTED** [review]

**Comment by delta1** [comment]
https://github.com/hansieodendaal/tari/pull/1

File: README.md (hunk 1 above)
Comment (delta1) [inline README.md]: `brew install cmake openssl tor ncurses coreutils`

**Comment by delta1** [comment]
Will test again first thing tomorrow 👍

**Review by delta1 — CHANGES_REQUESTED** [review]
Happy to approve (for macos) once hansieodendaal#2 is included

**Comment by hansieodendaal** [comment]
Thanks @delta1, will squash all commits now.

**Review by delta1 — APPROVED** [review]
LGTM

**Pull request merged**